pkg/image/controller/signature/container_image_downloader.go

```diff
@@ -26,6 +26,10 @@ func NewContainerImageSignatureDownloader(ctx context.Context, timeout time.Dura
 	}
 }
 
+type GetSignaturesError struct {
```

Comment:
deserves godoc

Reply:
no it doesn't.

Reply:
(what would it say?  "GetSignaturesError is an error returned when GetSignatures encounters an error"?)